test/sagaHelpers/takeLeading.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-unused-vars, no-constant-condition */
```

Comment:
personally i dont like disabling rules at the top of the file, its hard to determine why they were disabled

would prefer to mute rules at call sites, if possible